app/serializers/discussion_serializer.rb

```diff
@@ -44,26 +44,26 @@ def self.attributes_from_reader(*attrs)
   has_one :author, serializer: UserSerializer, root: 'users'
   has_one :group, serializer: GroupSerializer, root: 'groups'
   has_one :active_proposal, serializer: MotionSerializer, root: 'proposals'
+  has_one :active_proposal_vote, serializer: VoteSerializer, root: 'votes'
 
-  def author
-    object.author
+  def include_active_proposal_vote?
+    reader.present? && active_proposal.present?
+  end
+
+  def active_proposal_vote
+    active_proposal.votes.find_by(user_id: reader.user_id)
   end
 
   def active_proposal
-    object.current_motion
+    @active_proposal ||= object.current_motion
```

Comment:
I'm not quite following the @active_proposal bit because I can't see it defined anywhere. So it's an instance variable? Where would it be set/defined?